pages/linux/protontricks.md

```diff
@@ -23,6 +23,10 @@
 
 `protontricks -s {{game_name}}`
 
+- Run an executable in the proton environment of a specific game:
+
+`protontricks-launch --appid {{game_appid}} {{path/to/executable.exe}}`
```

Comment:
```suggestion
`protontricks-launch --appid {{appid}} {{path/to/executable.exe}}`
```
like in other examples

Reply:
I seems to have had a brainfart. Started typing "game_id", looked over to the other examples and wrote "game_appid" in the end.